index.html

```diff
@@ -46,778 +46,778 @@ <h3>Spring 2023, Class: Mon, Wed 4:30pm-5:50pm, Gates B1</h3>
+                                        Systems
+                                    </a> Rummery et al. (1994)
+                                </li>
+                                <li> <a href="https://arxiv.org/abs/1806.10293"> Flow Q-Learning
+                                    </a> Park et al. (2025)
+                                </li>
+                            </ul>
+                        </td>
+                    </tr>
+                    <tr class="active">
+                        <td><b>Week 3</b><br />
+                            Fri, April 18
+                        </td>
+                        <td><i><span class="label label-lecture text-base"> Lecture</span></i>
+                            <a">Practical Deep RL Implementation
+                                Techniques</a>
+                        </td>
+                        <td>
+                            <span class="label label-due text-base">Due</span></i><b> HW1 Due</b><br>
+                            HW 2 Out (DQN)
+                        </td>
+                        <td>
+                            <ul>
+                                <li> <a href="https://arxiv.org/abs/1509.06461"> Deep Reinforcement Learning
+                                        with Double Q-learning.</a> Hasselt et al. (2015)</li>
+                                <li> <a href="https://arxiv.org/abs/1806.10293"> A Distributional Perspective on
+                                        Reinforcement Learning.
+                                    </a> Bellemare et al. (2017)
+                                </li>
+                            </ul>
+                        </td>
+                    </tr>
+
+                    <tr>
+                        <td><b>Week 4</b><br />
+                            Wed, April 23
+                        </td>
+                        <td><i><span class="label label-lecture text-base">Lecture</span></i>
+                            <a">Offline RL</a>
+                        </td>
+                        <td>
+                        </td>
+                        <td>
+
+                            <ul>
+                                <li> <a href="https://arxiv.org/abs/2006.04779"> Conservative Q-Learning for Offline
+                                        Reinforcement Learning.
+                                    </a> Kumar et al. (2020)</li>
+                                <li> <a href="https://arxiv.org/abs/1906.08253"> Offline Reinforcement Learning with
+                                        Implicit Q-Learning </a> Kostrikov et al. (2021)</li>
+                            </ul>
+
+                        </td>
+                    </tr>
+
+                    <tr>
+                        <td><b>Week 4</b><br />
+                            Fri, April 25
+                        </td>
+                        <td><i><span class="label label-lecture text-base">Guest Lecture</span></i>
```

Comment:
This should be lecture not guest lecture. Can you make sure all of the guest lecture vs. lecture labels are correct?